package.json

```diff
@@ -61,7 +61,7 @@
   },
   "scripts": {
     "atlas": "node ./test/atlas_connectivity_tests.js",
-    "orchestration": "npm install -g shelljs && node test/check_mongo_orchestration_tests.js",
+    "orchestration": "rm -f package-lock.json && npm install -g shelljs && node test/check_mongo_orchestration_tests.js",
```

Comment:
We definitely shouldn't delete the `package-lock.json` at any point.

Reply:
I also don't think it's necessary to `npm install shelljs` if we're requiring it as a dependency.

Reply:
I added those two lines since I was getting the error: `cipm can only install packages when your package.json and package-lock.json or npm-shrinkwrap.json are in sync. Please update your lock file with `npm install` before continuing.` from the Travis tests on Node version 8 and 10
https://travis-ci.org/mongodb/node-mongodb-native/jobs/571502458

Reply:
You can do an `npm install` locally and that should correctly update the `package-lock.json`, which you can then commit to this PR. Forcibly removing the `package-lock.json` could lead to problems.

Reply:
Fixed!